token/js/src/extensions/transferHook/actions.ts

```diff
@@ -65,3 +75,156 @@ export async function updateTransferHook(
 
     return await sendAndConfirmTransaction(connection, transaction, [payer, ...signers], confirmOptions);
 }
+
+/**
+ * Add extra accounts needed for transfer hook to an instruction
+ *
+ * @param connection      Connection to use
+ * @param instruction     The transferChecked instruction to add accounts to
+ * @param commitment      Commitment to use
+ * @param programId       SPL Token program account
+ *
+ * @return Instruction to add to a transaction
+ */
+export async function addExtraAccountsToInstruction(
```

Comment:
nit: Can you move this function down to `instructions.ts`? The `actions` code lives above everything else, so moving this function will break a circular dependency

Reply:
Good catch!